pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Roll part of an application back to a previous deployment.
```

Comment:
Hrm, now I'm questioning "Roll".  "Revert part of an application back to a previous version through a deployment rollback."